src/app/business/block/cloud-file-share/create/cloud-file-share-create.component.ts

```diff
@@ -195,7 +216,32 @@ export class CloudFileShareCreateComponent implements OnInit{
             this.cloudFileShareCreateForm.addControl('metadata', this.fb.array([this.createMetadata('PerformanceMode', 'generalPurpose')]));
             this.addNextMetadata('ThroughputMode', 'bursting');
             this.addNextMetadata('ProvisionedThroughputInMibps', '');
-        } 
+        }
+        if(type == 'hw-file') {
+            this.cloudFileShareCreateForm.addControl('sfsType', new FormControl('SFS'));
+
+            if(this.cloudFileShareCreateForm.controls['sfsType']){
+                this.cloudFileShareCreateForm.get("sfsType").valueChanges.subscribe(
+                    (value:string)=>{
+                        this.sfsType = value
+                    }
+                );
+            }
+
+            this.cloudFileShareCreateForm.addControl('size', this.fb.control('1', [Validators.required]));
+            this.cloudFileShareCreateForm.addControl('availabilityZone', this.fb.control(''));
+            this.cloudFileShareCreateForm.addControl('encrypted', this.fb.control(false, [Validators.required]));
+            this.cloudFileShareCreateForm.addControl('protocols', this.fb.control('NFS'));
```

Comment:
Is `protocols` always going to be `NFS` ?
Can there be any other type of protocol?

Reply:
No Protocol will not be NFS always. NFS is added as a default protocol.
There can be other protocols like NFS, CIFS, SAMBA etc.
You can raise an issue to enhance the protocol field as a Drop Down menu

Reply:
In that case please let me know the possible values for the protocols that are supported as follows:
```
[
            {
                label: 'NFS',
                value: 'NFS'
            }
        ];
```
Are the values expected in uppercase or lowercase?
I will make the necessary changes and push them to your branch.

Reply:
Issue is opened for the enhancement for protocols selection through drop down menu.
We can enhance and track the FS enhancement support separately. 
https://github.com/sodafoundation/dashboard/issues/453